content/altinn-studio/reference/ux/texts/_index.nb.md

```diff
@@ -178,14 +178,18 @@ Variabler i tekster kan inkluderes ved å følge oppsettet nedenfor. Det er vikt
     {
       "key": "<instance value key>",
       "dataSource": "instanceContext"
+    },
+    {
+      "key": "<custom text parameter key>",
```

Comment:
This should probably also be `current`length` then? :face_with_peeking_eye:

Reply:
How about now?

Reply:
That also works! :raised_hands: